developers/prerequisites/Equinox.md

```diff
@@ -41,7 +41,7 @@ Then follow these steps:
 6. Click on "Validate Bundles" and make sure that "No problems were detected" is displayed.
 7. You can start Equinox with the "Run" button.
 
-<img src="images/RunConfiguration.png" width="600"/>  
+<img src="images/RunConfiguration.PNG" width="600"/>  
```

Comment:
Argh, no!
I would suggest to keep file names of images always lower case, so you should rather use the image file to runconfiguration.png.